src/leiningen/monolith.clj

```diff
@@ -181,13 +183,61 @@
   (checkouts/unlink project))
 
 
+;; ## Fingerprinting
+
+;; Leiningen's help framework doesn't officially support "subsubtasks" so
+;; we'll try our best to make it work by making them look like subtasks.
+
+(defn changed
+  "Show information about the projects that have changed since last :refresh.
+
+  Optionally takes one or more marker ids, or project selectors, to narrow the
+  information.
+
+  Usage:
+  lein monolith changed [project-selectors] [marker1 marker2 ...]"
+  [project args]
+  (let [[opts more] (u/parse-kw-args fingerprint/selection-opts args)
+        opts (u/globalize-opts project opts)]
+    (fingerprint/changed project opts more)))
+
+
+(defn mark
+  "Manually mark projects as refreshed.
```

Comment:
I'm concerned that `clear` and `mark` are really general names, so outside of a `lein monolith fingerprint ...` context these may not be as easy to understand or may conflict with other functionality in the future.